Makefile

```diff
@@ -21,6 +21,7 @@ HELM_OPTS ?= --set serviceAccount.annotations.eks\\.amazonaws\\.com/role-arn=${K
 			--set settings.aws.clusterEndpoint=${CLUSTER_ENDPOINT} \
 			--set settings.aws.defaultInstanceProfile=KarpenterNodeInstanceProfile-${CLUSTER_NAME} \
 			--set settings.aws.interruptionQueueName=${CLUSTER_NAME} \
+			--set crd-helm-chart-inclusion.enabled=true \
```

Comment:
```suggestion
			--set include-crds=true \
```

Reply:
Could we support the `--skip-cords` flag? https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

Reply:
I don't think so, I believe you can do that when you use the other option of putting CRDs in a directory like we were doing in past. Now that CRDs are dependencies that flag is ignored.